float64[6] setx